data/remnant/remnant missions.txt

```diff
@@ -1060,6 +1060,333 @@ mission "Remnant: Learn Sign Follow Up"
 			`	"Congratulations, Captain <last>!" the director signs. "Your fluency is good enough that even those unwilling to sing with a stranger should be comfortable working with you. At this rate you will be one of us in more ways than just by blood. I look forward to seeing what results from your work with us." The director is interrupted by the arrival of other Remnant asking for something and bids you farewell with an apologetic gesture.`
 				decline
 
+	
+mission "Remnant Rescue"
+	job
+	repeat
+	name "Search and Rescue: Scout"
+	description "The <npc> has gone missing during a scouting mission in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 30
+	npc accompany save
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+			not system "Edusa"
+		personality derelict timid plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant
+				Starling
+			variant
+				Gull
+	on complete
+		"rescue missions" ++
+		payment 200000
+		dialog
+			`Your communicator notifies you of a resource allocation of <payment>.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+
+# The Rescue Dummy is meant to suicide as soon as possible on entering the randomized system.
+ship "Rescue Dummy"
+	sprite "ship/behemoth"
+	attributes
+		"heat dissipation" 1
+		"hull" 1
+		"mass" 10
+		"drag" 1
+		"automaton" 1
+		"energy generation" 1
+		"thrust" 30
+		"turn" 30
+	outfits
+		"Suicide Gun"
+	gun 0 0
+
+outfit "Suicide Gun"
+	"category" guns
+	thumbnail "outfit/unknown"
+	"gun ports" -1
+	weapon
+		velocity 100
+		lifetime 10
+		reload 1
+		"blast radius" 900
+		"shield damage" 0.01
+		"hull damage" 1
```

Comment:
Yes. I agree, somebody mentioned the two ship thing, I just haven't gotten around to doing it. I definitely will though, as it sounds more reliable in so many ways (especially now that I have some idea of how to use these ships after the nightmare that was getting this to work).

As for the sprite used, I just wanted something noticable enough for when I was testing. I'll definitely get to work on making this new double-ship setup over the next few days, I like your idea of making a visible version of the ship that you're hunting for "just in case", because that also solves the issue of jumping too quickly (which *can* happen with the current setup, but it's very close).